tests/aws-cpp-sdk-core-tests/aws/auth/AWSCredentialsProviderTest.cpp

```diff
@@ -96,21 +96,25 @@ TEST_F(ProfileConfigFileAWSCredentialsProviderTest, TestDefaultConfig)
     credsFile << "[Somebody Else ]" << std::endl;
     credsFile << "aws_access_key_id = SomebodyElseAccessId" << std::endl;
     credsFile << "something else to break the parser" << std::endl;
+    credsFile << "aws_account_id=222222222222" << std::endl;
     credsFile << "#test comment" << std::endl;
     credsFile << "[default]" << std::endl;
     credsFile << "aws_access_key_id = DefaultAccessKey" << std::endl;
     credsFile << "aws_secret_access_key=DefaultSecretKey " << std::endl;
     credsFile << "aws_session_token=DefaultSessionToken" << std::endl;
+    credsFile << "aws_account_id=111111111111" << std::endl;
     credsFile << std::endl;
     credsFile << " [Somebody Else Again]" << std::endl;
     credsFile << "aws_secret_access_key = SomebodyElseAgainAccessId" << std::endl;
-    credsFile << " aws_secret_access_key=SomebodyElseAgainSecretKey" << std::endl;
```

Comment:
please keep a random white space in the test credentials file as it is a valid for credentials file to have random spaces here and we must still be able to read it.

Although it should be a different test for the random file formatting.

Reply:
sure